pkg/registry/apps/deployment/storage/storage_test.go

```diff
@@ -374,6 +379,31 @@ func TestEtcdCreateDeploymentRollback(t *testing.T) {
 	}
 }
 
+func TestCreateDeploymentRollbackValidation(t *testing.T) {
+	storage, server := newStorage(t)
+	rollbackStorage := storage.Rollback
+	rollback := apps.DeploymentRollback{
+		Name:               name,
+		UpdatedAnnotations: map[string]string{},
+		RollbackTo:         apps.RollbackConfig{Revision: 1},
+	}
+
+	ctx := genericapirequest.WithNamespace(genericapirequest.NewContext(), namespace)
+
+	if _, err := storage.Deployment.Create(ctx, validNewDeployment(), rest.ValidateAllObjectFunc, &metav1.CreateOptions{}); err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	_, err := rollbackStorage.Create(ctx, &rollback, denyCreateValidation, &metav1.CreateOptions{})
+
+	if err == nil {
+		t.Error("Expected admission error but got nothing")
```

Comment:
>  	return fmt.Errorf("admission denied")

verify returned error

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/72271/commits/0293c9cc218931dd3568e93c394febad03179497